lib/collection/query-param.js

```diff
@@ -136,7 +138,7 @@ _.extend(QueryParam, /** @lends QueryParam */ {
         if (!obj) { return ''; }
         var unparsed = obj.key || '';
         // if the value is not null, then we add the value
-        (obj.value != null) && (unparsed += ('=' + (obj.value || '')));
+        (obj.value !== null) && (obj.value !== undefined) && (unparsed += ('=' + (obj.value || '')));
```

Comment:
No changes in behavior please. Add a `todo` if you think this is incorrect, and we'll take it up in a future PR.